scripts/generate-opt-in-redirect-checks.ts

```diff
@@ -10,6 +10,12 @@ const filePath = path.join(
 	'opt-in-redirect-checks.js'
 )
 
+/**
+ * This generates {@link [../src/.generated/opt-in-redirect-checks.js](../src/.generated/opt-in-redirect-checks.js)}
+ * which is .gitignore'd, so it may not appear in grep results.
+ *
+ * This will contain a dictionary of `{[productslug]: RegExp}`
+ */
```

Comment:
A Clickable link in JSDoc comment:
<img width="819" alt="CleanShot 2022-10-14 at 15 49 01@2x" src="https://user-images.githubusercontent.com/26389321/195929920-9d479335-6952-4a20-adfd-0d55cb35fedb.png">